wiki/entry_hahn.qmd

```diff
@@ -0,0 +1,32 @@
+---
+title: "Wiki Entry Template"
+ subtitle: "BA New Media / Online Communication KF/BF A: Spring Term 2023"
+ author: "Alina Hahn"
+ categories:
+- "Digital Media"
+---
+
+
+Wiki Project: What Constitutes Autonomy in Digital Communication?
+
+Digital media use is one of the most talked about topics at the moment and can be linked to different aspects of research. One of the most common areas of research is digital media use and its impact on mental health. There is a wide range of research possibilities and in the following paper the link between digital media use and mental health is explained based on the article ´Computer-Mediated Communication, Social Media, and Mental Health: A Conceptual and Empirical Meta-Review´ by Reinecke and Meier (2021). The authors focus not only on digital media use, but also extend the research to include autonomy needs and computer-mediated communication (CMC).
+
+´Computer-mediated communication (CMC), the Internet, and now social and mobile media have repeatedly been characterised as a blessing or a curse for users’ mental health (MH)´ (p.1183) is the main aspect of the paper. The authors explain the relationship between social media use and mental health with negative and positive examples and highlight the importance of the research area. When looking at the relationship between computer-mediated communication and mental health it is clear, that the two can’t go without each other. The authors argue the importance of addressing both channel-centric and communication-centric factors in the relationship between CMC and mental health. The authors show a CMC taxonomy that differentiates between six levels. They also propose two operational approaches: a technology-centered approach that emphasises technology characteristics, and a user-centered approach that focuses on user behaviours and interactions. These frameworks can help to research more about the complex relationship between CMC, social media, and mental health.
```

Comment:
What are these different six levels of CMC taxonomy? You mention these six levels, but doesnt name it or explain these. So I wonder waht these are.